release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -689,6 +740,33 @@ This dashboard provides API server metrics, such as:
 
 // TODO: Link to troubleshooting using this dashboard section once it is available
 
+[id="ocp-4-7-monitoring-namespace-by-pod-and-pod-total-Grafana-dashboards"]
+==== New Namespace By Pod and Pod Total monitoring dashboards
+
+The *Namespace By Pod* and *Pod Total* networking Grafana dashboards are now enabled. You can access the *Namespace By Pod* and *Pod Total* dashboards from the web console by navigating to *Monitoring* -> *Dashboards* and selecting either dashboard.
```

Comment:
@dgrisonnet Can you please provide some details on what the newly enabled Namespace By Pod and Pod Total Grafana dashboards are used for? Thanks! (See the "New API Performance monitoring dashboard" section above this note as an example)